backend/segmentation/models/frap_select.py

```
@@ -185,28 +186,54 @@ def _slice_window(pts: np.ndarray, i: int, half_px: float, step_px: float) -> np
+    while its unit tests passed on synthetic data an order of magnitude brighter
+    than any microscope produces.
+
+    In units of the ring's spread the same 19 filaments read a median of 3.8
+    sigma on TIRF_491 and 3.4 on TIRF_561, so a 2 sigma cut admits 15 of 19 —
+    and TIRF_642, which carries no filament signal at all, still reads 0.19
+    sigma and is still rejected 19 times out of 19. The criterion discriminates;
+    it was only ever on an unreachable scale.
     """
     h, w = fluor.shape[:2]
     band = mt_measure.rasterize_band(window.astype(np.float32), h, w, p.band_thickness_px)
     margin_radius = int(round(p.band_thickness_px * p.margin_multiplier))
```

Ruff (0.16.2)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)